lib/tic_tac_toe_web_app.rb

```diff
@@ -0,0 +1,34 @@
+require 'sinatra/base'
+require 'board'
+require 'player_symbols'
+require 'one_indexed_grid_formatter'
+require 'web_display_to_board_adapter'
+require 'web_game'
+
+class TicTacToeWebApp < Sinatra::Base
+
+  set :views, File.dirname(__FILE__) + '/../views'
+  set :public_folder, File.dirname(__FILE__) + '/../public'
+
+  get '/' do
+    @formatted_rows = OneIndexedGridFormatter::new.format(Board.new)
+    @valid_moves = PlayerSymbols::all
+    erb :landing_page
+  end
+
+  get '/next_move' do
```

Comment:
I would split 'web stuff'  from 'ttt stuff'.
So I'd expect `get '/next_move` to to extract everything it needs from the request,
hand that data over and convert the result to something 'web-ish' like HTML or a redirect.